### DISCUSSION TITLE
How do you use GitOps?

### DISCUSSION THREAD

**Discussion by scottrigby** [open-gitops/project#25] [Q&A]
## Hi everyone! 👋  How do you use GitOps?

Please comment on this discussion topic to let the community know how you're using GitOps, and any important considerations you would like to mention. These can also help us develop use cases for a near future "GitOps Use Cases" supporting document in https://github.com/open-gitops/documents.

You may also optionally request to discuss and/or demos your comments here by planning to attend and add to the GitOps Working Group meetings agenda: <https://github.com/cncf/tag-app-delivery/tree/master/gitops-wg>

Thanks for contributing! 🙇 🦄 👀

**Comment by scottrigby** [comment]
Too general of a question for the goal?

**Reply by johhess40** [reply]
@scottrigby I don't personally think it's too open ended. I think that the practice of GitOps can be applied in a wide variety of use cases even though some may be more prevalent than others. For example, one could even apply the GitOps principles to creating documentation for a project.

**Comment by roberthstrand** [comment]
Usually, it's how the clients that I work with maintain their applications. I'm currently working on a MSP-like scenario where GitOps will be very integral to the process, which hopefully can help us manage both infrastructure and application delivery in a standardized way.
